src/komal/coding_exam_1/ReturnIndices.java

```diff
@@ -0,0 +1,39 @@
+package komal.coding_exam_1;
+
+import java.util.Scanner;
+
+/*
+ * Program 1 : 
+Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+Input: nums = [2,11,15,7,4], target = 9
+Output: [0,3]
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+ */
+public class ReturnIndices {
+
+		int num;
+
+		void indices(int[] array, int target) {
+			for (int i = 0; i < array.length; i++) {
+				for (int j = 1; j < array.length; j++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files